handbook/product/roles/product_designer.md

```diff
@@ -38,6 +38,8 @@ We will encourage and support you to:
 - Experience working in cross-functional teams on multiple projects in tandem.
 - Experience building or contributing to the design of consumer or enterprise applications.
 - Experience working with distributed or remote teams.
+- Experience builing and developing complex design systems.
+- Experience working on or leading big product redesign.
```

Comment:
Want to clarify it's about the visual design more than how the product "works," e.g. that it's a visual overhaul rather than a rebuild from scratch.

```suggestion
- Experience working on or leading comprehensive design renewal for consumer or enterprise products.
```

Reply:
I agree... 'consumer' is not accurate (I realize it was here before!) and we are looking for strong UI design more than specific redesign experience.

Reply:
Maybe we can change it to 'Strong UI design skills. Nice to have: experience working on or leading a product redesign' @rrhyne @quinnkeast @christinaforney